content/en/account_management/org_settings/sensitive_data_detection.md

```diff
@@ -24,31 +24,31 @@ Often businesses are required to identify, remediate, and prevent the exposure o
 
 ## Sensitive Data Scanner
 
-Sensitive data scanner is a stream-based, pattern matching service that you can use to identify, tag, and optionally redact or hash sensitive data. With implementation, your security and compliance teams can introduce a new line of defense in preventing sensitive data from leaking outside your organization.
+Sensitive Data Scanner is a stream-based, pattern matching service that you can use to identify, tag, and optionally redact or hash sensitive data. With implementation, your security and compliance teams can introduce a new line of defense in preventing sensitive data from leaking outside your organization.
 
-Sensitive data scanner can be found under [Organization Settings][1]. The Scanner Library offers a wide variety of rules for commonly detected patterns such as email addresses, credit card numbers, API keys, authorization tokens, and others.
+Sensitive Data Scanner can be found under [Organization Settings][1].
 
 {{< img src="logs/sensitive_data_scanner/sensitive_data_scanner3.png" alt="Sensitive Data Scanner in Organization Settings" style="width:90%;">}}
```

Comment:
Is this image supposed to be replaced by sds_main_apr_22.png?

Reply:
Updated